io/aio-wrapper.cpp

```diff
@@ -166,7 +166,7 @@ namespace photon
                          VALUE(piocb->u.c.buf), VALUE(piocb->u.c.resfd));
             thread_interrupt((thread *)events[i].data, EOK);
         }
-        if (n == IODEPTH)
+        if (n == IODEPTH_MAX)
```

Comment:
should use iodepth here instead of IODEPTH_MAX

Reply:
Here always collect at most IODEPTH_MAX events
so the maximal n should equal to event array size and the parameter N calling `my_io_getevents`,
It should be still IODEPTH_MAX

Reply:
if iodepth < IODEPTH_MAX, n always less than IODEPTH_MAX